code/_onclick/item_attack.dm

```diff
@@ -60,11 +60,15 @@
 	SEND_SIGNAL(M, COMSIG_MOB_ITEM_ATTACKBY, user, src)
 
 	var/nonharmfulhit = FALSE
-	if(item_flags & NOBLUDGEON)
-		nonharmfulhit = TRUE
 
 	if(user.a_intent == INTENT_HELP && !(item_flags & ISWEAPON))
 		nonharmfulhit = TRUE
+	for(var/datum/surgery/S in M.surgeries)
+		if(S.failed_step)
```

Comment:
If there are multiple surgeries on a mob, and one of those were failed, it will return TRUE.
It should check if the aim and the surgery target are the same

Reply:
I used another method.